ietf-te-path-computation.yang

```diff
@@ -497,27 +535,315 @@ module ietf-te-path-computation {
   }
 
   /*
-   * AUGMENTS TO TE RPC
+   * Augment TE RPCs
    */
 
-  augment "/te:tunnels-rpc/te:input/te:tunnel-info" {
+  augment "/te:tunnels-path-compute/te:input/te:path-compute-info" {
     description "Path Computation RPC input";
     list path-request {
       key "request-id";
-      description "request-list";
+      description "The list of the requested paths to be computed";
       leaf request-id {
         type uint32;
         mandatory true;
         description
           "Each path computation request is uniquely identified
            by the request-id-number.";
```

Comment:
The request-id-number seems to be provided by the client. What is the scope in which the request-id-number is unique? Is it unique within a NETCONF session for example?

Reply:
Addressed in https://github.com/rvilalta/ietf-te-path-computation/pull/70/commits/7cf3781791607c49b0795b4a7b892ca99289a6a3